routes/web.php

```diff
@@ -11,6 +11,10 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+//Route::get('/', function () {
```

Comment:
You might want to remove this commented out code. This helps you to keep your code clean and tidy.
If you need to comment out code, always explain why it is commented. That really helps other developers understanding why you did it :)